docs/translations.md

```diff
@@ -34,4 +34,4 @@ We're collecting [tips](https://github.com/github/opensource.guide/issues/1119)
 
 ### New articles
 
-New articles are rare! When we have one, we'll probably do [some form](https://github.com/github/opensource.guide/issues/1120) of a call for translations.
\ No newline at end of file
+New articles are rare! When we have one, we'll probably do [some form](https://github.com/github/opensource.guide/issues/1120) of a call for translations.
```

Comment:
What changed here?

Reply:
I didn't made any change here, I think there may be a shift in space

Reply:
Ok, np!